### PR TITLE
Add a test_ticket tag to new ZD tickets created outside of prod

### PR DESCRIPTION
**Pull request by jenny-heath**

### CODE REVIEW

File: spec/services/zendesk_intake_service_spec.rb [hunk 1]
```diff
@@ -59,8 +59,10 @@
     DatadogApi.configure do |c|
       c.enabled = true
       c.namespace = "test.dogapi"
+      c.env = "production"
```
Comment (bengolder) [inline spec/services/zendesk_intake_service_spec.rb:62]: Why was this line added?

Comment (bengolder) [reply spec/services/zendesk_intake_service_spec.rb:62]: Was it needed to prevent breakage of the test?

Comment (jenny-heath) [reply spec/services/zendesk_intake_service_spec.rb:62]: yes! because the test_ticket tag depends on the environment, i had to change all the places where env was set in the test to be consistent otherwise the test failed. it was actually a bit of a challenge because i didn't realize i had to unset it in the after block
